lib/cide/cli.rb

```diff
@@ -123,8 +123,8 @@ def exec
       type: :boolean,
       default: true
 
-    method_option 'set_version',
-      desc: 'Tells cide the package version, otherwise extracted from git',
+    method_option 'set_build_id',
```

Comment:
the `set` prefix is not needed anymore, it was added to disambiguate with `--version` which usually returns the program version